### DISCUSSION TITLE
Allow x-frames

### DISCUSSION THREAD

**Discussion by Countryman98** [AzuraCast/AzuraCast#5194] [General]
Have had a look around but cant find any info, Where in Azuracast can you set x-frame-options?

I Would like to be able to have my Azuracast podcast page inside an Iframe on my Station domain which is not the same domain as my Azuracast instance resulting in "Refused to display 'https://azuracast.zen/' in a frame because it set 'X-Frame-Options' to 'deny'."

[AzuraCast](https://www.azuracast.com/) • Rolling Release #[aa5a3fe] • Docker 

Thanks

**Comment by punyauk** [comment]
Did you find an answer to this as I have same issue?
